ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarDemoController.swift

```diff
@@ -106,6 +106,8 @@ class AvatarDemoController: DemoController {
             let overflowAvatar = createAvatarView(size: size,
                                                   name: "20",
                                                   style: .overflow)
+            overflowAvatar.state.accessibilityLabel = "20 more contacts"
```

Comment:
does the client needs to add accessibility label for all the avatars? What's the default value?

Reply:
This is a customization scenario where the client App could set "Signed Out" as the accessibility label for displaying an avatar with no data.
Or as in this example, instead of showing just "20", it provides a more meaningful description of what it means. ("20 more contacts" for example)